helm-values/tenant-ui/values-sandbox.yaml

```diff
@@ -29,15 +35,15 @@ oidc:
   active: false
   showInnkeeperAdminLogin: false
   showWritableComponents: false
```

Comment:
Based on the [comment](https://github.com/bcgov/trust-over-ip-configurations/pull/146#discussion_r1320308853) by @loneil these actually need to be set to `true`, please.